src/landing.tsx

```diff
@@ -0,0 +1,213 @@
+// import courseKataLogo from '../style/icons/landing/coursekata.svg';
+// import skewTheScriptLogo from '../style/icons/landing/skewthescript.svg';
+
+// TODO: find a better way to handle these imports
+import '../style/landing.css';
+import '../style/base.css';
+
+/**
+ * LandingPage component renders the landing page for Jupyter Everywhere.
+ * It includes a header, hero section with buttons to create notebooks,
+ * a features section, a testimonial, an about section, and a footer.
+ * @returns Landing page component
+ */
+function LandingPage(): JSX.Element {
+  const handleCreateNotebook = (kernel: string) => {
+    window.location.href = `/?notebook=Untitled.ipynb&kernel=${kernel}`;
+  };
+
+  // Placeholder function for upload functionality
+  const handleUpload = () => {
+    alert('Upload functionality not yet implemented.');
+  };
+
+  const handleScrollToFeatures = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    const featuresSection = document.getElementById('features');
+    if (featuresSection) {
+      featuresSection.scrollIntoView({
+        behavior: 'smooth',
+        block: 'start'
+      });
+    }
+  };
+
+  const handleCardKeyDown = (e: React.KeyboardEvent, kernel: string) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.preventDefault();
+      handleCreateNotebook(kernel);
+    }
+  };
+
+  return (
+    <div className="je-landing">
+      {/* Hero section */}
+      <section className="je-hero">
+        <div className="je-header">
+          <img src={jeOctopus} className="je-logo" alt="Logo" />
+          <button className="je-tutorial-button">Tutorial</button>
+        </div>
+
+        <main className="je-main">
+          <h2 className="je-brand">Welcome to</h2>
+          <h1>
+            <span className="je-brand">Jupyter Everywhere</span>
+          </h1>
+
+          <div className="je-buttons">
+            <div
+              className="je-card"
+              onClick={() => handleCreateNotebook('python')}
+              onKeyDown={e => handleCardKeyDown(e, 'python')}
+              tabIndex={0}
+              role="button"
+              aria-label="Create Python Notebook"
+            >
+              <p>Create Python Notebook</p>
+              <img src={pythonLogo} alt="Python logo" />
+            </div>
+
+            <div
+              className="je-card"
+              onClick={() => handleCreateNotebook('xr')}
+              onKeyDown={e => handleCardKeyDown(e, 'xr')}
+              tabIndex={0}
+              role="button"
+              aria-label="Create R Notebook"
+            >
+              <p>Create R Notebook</p>
+              <img src={rLogo} alt="R logo" />
+            </div>
+          </div>
+          <a
+            href="#"
+            className="je-upload"
+            onClick={e => {
+              e.preventDefault();
+              handleUpload();
+            }}
+          >
+            Upload a notebook
```

Comment:
Uppercase on design:

```suggestion
            Upload a Notebook
```